pkgs/by-name/li/libbgcode/package.nix

```
@@ -7,17 +7,17 @@
   heatshrink,
   zlib,
   boost,
-  catch2,
+  catch2_3,
 }:
 stdenv.mkDerivation {
   pname = "libbgcode";
-  version = "2023-11-16";
+  version = "2025-02-19";
```

```suggestion
  version = "0-unstable-2025-02-19";
```
To follow the usual convention for unreleased packages